app/models/maestrano/connector/rails/external.rb

```diff
@@ -15,6 +15,6 @@ def self.create_account_link(organization = nil)
   end
 
   def self.entities_list
-    %w(organization)
+    %w(organization deal product)
```

Comment:
Just as a remainder for the next PR, the order here is important for references. Please add 'app_users' at the beginning of the list, and put product before deal. 
```ruby
%w(app_user organization product deal)
```